### ISSUE TITLE
Error during topic parsing

### ISSUE THREAD

**Issue by stavenko** [stavenko/forum-loader#1]
Processing board #29/120: Question: Hashgraph
        Processing Page 1/0
        Processing board #30/120: Guidance: Implementing a private and custom blockchain solution for supply chain
TypeError: Cannot read property 'querySelectorAll' of undefined
    at getAllMessagesFromTopicPage (/root/forum/forum-loader-master/index.js:135:5)
    at retrieveAllMessages (/root/forum/forum-loader-master/index.js:119:20)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

**Comment by stavenko** [comment]
Не смог воспроизвести - видимо, тот форум был удален или перенесен. Добавил ключик, чтобы начинать скачивание с определенного номера борды. Например:

`node ./index.js -b 30`

**Comment by stavenko** [comment]
Topic list before having error
``` 
2FA Removal HitBTC
Get list of all addresses with a balance over x?
DagCoin: a cryptocurrency without blocks
c-lightning on testnet: problems and questions
Question: Hashgraph
```

**Comment by stavenko** [comment]
Error caused by forum 503 error. fixed with 5c01ab2b9faffadb commit
